specifyweb/notifications/tests.py

```diff
@@ -34,4 +34,30 @@ def test_get_notification_with_param_since(self):
         self.assertEqual(mockResponse[0]['message_id'], responseReturned[0]['message_id'])
         self.assertEqual(mockResponse[0]['read'], responseReturned[0]['read'])
         self.assertEqual(mockResponse[0]['timestamp'][:-7], responseReturned[0]['timestamp'][:-7])
+        
+    def test_delete_all(self):
+        # Create some test messages
+        testMessage1 = Message.objects.create(
+            user=self.specifyuser,
+            timestampcreated=datetime.now(),
+            content=json.dumps({'type': 'test-type-1', 'file': 'test_file_1.csv'})
+        )
+        testMessage2 = Message.objects.create(
+            user=self.specifyuser,
+            timestampcreated=datetime.now(),
+            content=json.dumps({'type': 'test-type-2', 'file': 'test_file_2.csv'})
+        )
+
+        c = Client()
+        c.force_login(self.specifyuser)
+
+        # Send a POST request to the delete_all endpoint with the IDs of the test messages
+        response = c.post('/notifications/delete_all/', {'message_ids': json.dumps([testMessage1.id, testMessage2.id])})
+
+        # Check if the response is OK
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.content.decode(), 'OK')
 
+        # Check if the messages have been deleted
+        self.assertFalse(Message.objects.filter(id=testMessage1.id).exists())
+        self.assertFalse(Message.objects.filter(id=testMessage2.id).exists())
```

Comment:
(optional) 
If you want to only hit the database once after the API has been called, this can be simplified to 

```suggestion
        self.assertFalse(Message.objects.filter(id__in=[testMessage1.id, testMessage2.id]).exists())
```
If you wanted to make this approach even better, you could further extract `[testMessage1.id, testMessage2.id]` into a separate variable and use it in the `self.assertFalse` and when making the request to the endpoint https://github.com/specify/specify7/blob/360a1832cadf787ff37d15e98f4f671b9a23e6e5/specifyweb/notifications/tests.py#L55

(Extracting the ids into a variable would be helpful if this test would ever need to be changed. Instead of modifying every place which needs the ids, only the variable would need to be modified. Additionally, it makes the code more readable. Places which use the message id array don't need to know excactly which ids they are using, just that they are using the array of message ids, i.e., `messageIds` over `[testMessage1.id, testMessage2.id, ...]`)